packages/connect/src/data/__tests__/defaultFeeLevels.test.ts

```diff
@@ -80,9 +96,10 @@ describe('getEthereumFeeLevels', () => {
         },
     };
 
-    Object.entries(fixtures).forEach(([chain, { expected }]) => {
+    Object.entries(fixtures).forEach(([chain, { expected, coinInfo }]) => {
         it(`should return correct fee levels for ${chain}`, () => {
-            const result = getEthereumFeeLevels(chain);
+            // @ts-expect-error
+            const result = getEthereumFeeLevels(coinInfo);
```

Comment:
```suggestion
            const result = getEthereumFeeLevels(chain, coinInfo);
```
Should fix unit test